crates/kernel/testsuite/common/mod.rs

```diff
@@ -102,17 +103,21 @@ pub fn run_as_verb(db: Arc<dyn WorldStateSource>, expression: &str) -> ExecResul
     result
 }
 
-pub fn eval(db: Arc<dyn WorldStateSource>, player: Objid, expression: &str) -> ExecResult {
-    let binary = compile(expression).unwrap();
-    let mut state = db.new_world_state().unwrap();
+pub fn eval(
+    db: Arc<dyn WorldStateSource>,
+    player: Objid,
+    expression: &str,
+) -> eyre::Result<ExecResult> {
+    let binary = compile(expression)?;
+    let mut state = db.new_world_state()?;
     let result = call_eval_builtin(
         state.as_mut(),
         Arc::new(NoopClientSession::new()),
```

Comment:
not in scope for this review, but i wonder if we should provide a client session impl here that does logging & log capture of some kind.

Reply:
Logging capture as in, `tracing`? Wouldn't that just need a sane `tracing` config for tests?
Or logging as in, the commands sent / responses received?

When I first saw this, I wondered if this is related to user management but probably not?

Oh, oh, also, is *this* the bit where a real implementation would allow `; obj = create(#2);` to retain `obj` for future commands?

Reply:
The client session interface trait is for anything that can produce output (and receive input). So for things that happen in-moo world (notify, connect, disconnect, etc) can output over that interface.

https://github.com/rdaum/moor/blob/main/crates/kernel/src/tasks/sessions.rs

The noop implementation just throws all that output away.